src/terra-dev-site/app/components.2/ApplicationEventEmitter.app.mdx

```diff
@@ -0,0 +1,159 @@
+# Event Emitter
```

Comment:
I propose that event emitter should fall into the utilities section rather than components.

My reasoning is that components are react components and utilities are the extra stuff.

Reply:
That is a valid point. https://github.com/cerner/terra-application/pull/72/commits/30371e4bdb89d1e60da4cdf0f3e5dccbb9333544

Should we keep the name `ApplicationEventEmitter` or simply name it `EventEmitter` similar to us naming Logger instead of ApplicationLogger?